build-tools/heroku/tailwind/styles.css

```diff
@@ -27,6 +27,7 @@ h4 {
   @apply font-bold text-base;
 }
 
+/* TB: Why do we have this? It makes it (a lot) harder to style these element within the templates... */
```

Comment:
@TiBiBa Do we need an issue so we don't forget to look into this? (I also don't remember how this came to do)